src/main/python/hydra/lib/hydrabase.py

```diff
@@ -77,17 +79,45 @@ def _execute(self, method, **kwargs):
             func(**kwargs)
             ha.stop()
 
-    def _get_group_info(self):
-        """
-        Return group info
-        """
-        return self.group_info
+    def add_tasks_to_group(self, tasks_list):
+        """
+        Add tasks to the group.
+        :param tasks_list: Tasks list to add. Duplicate tasks will be added only once.
+        :return: 0 in case of success.
```

Comment:
In current workflow, duplicate tasks are not possible. However, function is not making any assumption so it is handling duplicate tasks. 
Moreover, currently we dont have any requirement in our test cases to add tasks to existing group directly. So, we can remove the duplicate check. However, some test case might need this in functionality in future. In that case, this check is necessary.